src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs

```diff
@@ -104,6 +104,11 @@ internal class KafkaListener<TKey, TValue> : IListener, IScaleMonitorProvider
                 builder.SetValueDeserializer(ValueDeserializer);
             }
 
+            builder.SetLogHandler((_, m) =>
+            {
+                logger.LogInformation($"Libkafka: {m?.Message}");
```

Comment:
Confluent.Kafka.LogMessage has a log level that we could map to ours in order to have errors as errors, etc.

Reply:
Done